tests/cli/test_cli.py

```diff
@@ -1077,26 +1077,47 @@ def test(ctx):
     check_contains('non_root_user', _run_command([cl, 'uls']))
 
 
-@TestModule.register('freeze')
-def test_freeze(ctx):
+@TestModule.register('worksheet_freeze_unfreeze')
+def test_worksheet_freeze_unfreeze(ctx):
     _run_command([cl, 'work', '-u'])
     wname = random_name()
     wuuid = _run_command([cl, 'new', wname])
     ctx.collect_worksheet(wuuid)
     check_contains(['Switched', wname, wuuid], _run_command([cl, 'work', wuuid]))
-    # Before freezing: can modify everything
+    # Before freezing: can modify everything on the worksheet
     uuid1 = _run_command([cl, 'upload', '-c', 'hello'])
     _run_command([cl, 'add', 'text', 'message'])
     _run_command([cl, 'wedit', '-t', 'new_title'])
     _run_command([cl, 'wperm', wuuid, 'public', 'n'])
+
     _run_command([cl, 'wedit', '--freeze'])
-    # After freezing: can only modify contents
+
+    # After freezing: cannot modify anything
     _run_command([cl, 'detach', uuid1], 1)  # would remove an item
     _run_command([cl, 'rm', uuid1], 1)  # would remove an item
     _run_command([cl, 'add', 'text', 'message'], 1)  # would add an item
+    _run_command([cl, 'wedit', '-t', 'new_title'], 1)  # would edit
+    _run_command([cl, 'wperm', wuuid, 'public', 'a'], 1)  # would edit
+
+    _run_command([cl, 'wedit', '--unfreeze'])
+
+    # After unfreezing: can modify everything
+    _run_command([cl, 'detach', uuid1])  # would remove an item
+    _run_command([cl, 'rm', uuid1])  # would remove an item
+    _run_command([cl, 'add', 'text', 'message'])  # would add an item
     _run_command([cl, 'wedit', '-t', 'new_title'])  # can edit
     _run_command([cl, 'wperm', wuuid, 'public', 'a'])  # can edit
 
+    # Verify that we can make multiple edits to a frozen worksheet,
```

Comment:
I think what this code is really verifying is that "we can make a single edit to a frozen edit, as long as we unfreeze at the same time" -- can we update the comment accordingly?

Reply:
@nelson-liu I just slightly updated the tests for clarity regarding this comment; let me know if you have any concerns!

Reply:
Oops sorry, I must have missed this---you're correct, the new comment is more accurate. Thanks for following up!